python/hawkey/sack-py.cpp

```diff
@@ -226,15 +233,16 @@ sack_init(_SackObject *self, PyObject *args, PyObject *kwds)
     PyObject *logfile_py = NULL;
     self->log_out = NULL;
     int make_cache_dir = 0;
+    bool debug = false;
     gboolean all_arch = FALSE;
     const char *kwlist[] = {"cachedir", "arch", "rootdir", "pkgcls",
-                      "pkginitval", "make_cache_dir", "logfile", "all_arch",
-                      NULL};
+                      "pkginitval", "make_cache_dir", "logfile", "logdebug",
+                      "all_arch", NULL};
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|OssOOiOi", (char**) kwlist,
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|OssOOiOpi", (char**) kwlist,
```

Comment:
`p` - New in version 3.3. We want support for older Python.
Tips:
Pass the argument as object `O`. Or object with type control `O!`.
Then use `PyObject_IsTrue(pythonObject)`. Eg. `to_query` in `nevra-py.cpp`.

Reply:
I see. How about using `"i"` then, as it's used here for e.g. `"all_arch"`? Or do you find `"O"` better for some reason?

Reply:
I've changed it to `"O!"`.